databricks/koalas/tests/test_dataframe.py

```diff
@@ -3717,3 +3717,15 @@ def test_iteritems(self):
 
         for p_items, k_items in zip(pdf.iteritems(), kdf.iteritems()):
             self.assert_eq(repr(p_items), repr(k_items))
+
+    def test_tail(self):
+        if LooseVersion(pyspark.__version__) >= LooseVersion("3.0"):
+            pdf = pd.DataFrame(range(1000))
+            kdf = ks.from_pandas(pdf)
+
+            self.assert_eq(repr(pdf.tail()), repr(kdf.tail()))
+            self.assert_eq(repr(pdf.tail(10)), repr(kdf.tail(10)))
+            self.assert_eq(repr(pdf.tail(-990)), repr(kdf.tail(-990)))
+            self.assert_eq(repr(pdf.tail(0)), repr(kdf.tail(0)))
```

Comment:
Why do we using `repr`?

Reply:
These tests are failed in my local without `repr` like the below.
(Even though they looks same to me.)

```
E   AssertionError: False is not true : DataFrames are not equal:
E
E   Left:
E          0
E   995  995
E   996  996
E   997  997
E   998  998
E   999  999
E   0    int64
E   dtype: object
E
E   Right:
E          0
E   995  995
E   996  996
E   997  997
E   998  998
E   999  999
E   0    int64
E   dtype: object
```

Maybe should it better to use `almost=True` rather `repr` ?

Reply:
Maybe the column name of pdf is numeric instead of string.

Reply:
@ueshin Oh, It makes sense. Thanks.